### PR TITLE
Add Experimental SimpleFIN Sync to Docs

### PR DESCRIPTION
**Pull request by duplaja**
Added a new page in the docs under "Experimental", for SimpleFIN Sync. 

I used the existing GoCardless page as a starting point.

### CODE REVIEW

**Comment by duplaja** [comment]
Looks like I have a couple of spellings I missed. Will fix try here again.

File: docs/experimental/simplefin-sync.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+5. When you've mapped all your accounts that you wish to, click the "Link Account" button.
+
+6. To sync all accounts, click on "All Accounts" in the side menu, and then click "Sync", below the "All Accounts" header.
+
+### SimpleFIN Considerations
+
+1. Currently, the first sync pulls at most 90 days of data from each linked account. The amount of data SimpleFIN can get from each account may vary, so not all accounts may have 90 days of data that can be imported.
+
+2. Subsequent syncs are based off of the last succesful sync, with a small lookback period.
```
Comment (youngcw) [inline docs/experimental/simplefin-sync.md:86]: I think, as of now, it will always pull 90 days if it can.

Comment (duplaja) [reply docs/experimental/simplefin-sync.md]: Thank you, I had misremembered that. I've gone ahead and updated it.

**Comment by duplaja** [comment]
Not quite sure what I need to do here for the spelling check, sorry.

File: .github/actions/spelling/expect.txt [hunk 2]
```diff
@@ -2,7 +2,6 @@ AESUDEF
 Belfius
 Biedenkopf
 BNP
-caddy
```
Comment (youngcw) [inline .github/actions/spelling/expect.txt:5]: is there a reason you removed this?

Comment (duplaja) [reply .github/actions/spelling/expect.txt:5]: Not directly, no. It was listed as "previously acknowledged words that are now absent", by the Check Spelling check (see: https://github.com/actualbudget/docs/pull/312#issuecomment-1931019518 ).

There was a command listed (in that same message) to accept unrecognized words. 

`curl -s -S -L 'https://raw.githubusercontent.com/check-spelling/check-spelling/main/apply.pl' |
perl - 'https://github.com/actualbudget/docs/actions/runs/7808058390/attempts/1'`

Didn't realize I could modify that expect.txt file directly.

**Review by youngcw — APPROVED** [review]

**Pull request merged**